CHANGELOG.md

```diff
@@ -54,6 +54,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Don't release vts explicitly. [#261](https://github.com/greenbone/ospd-openvas/pull/261)
 - Drop handling of network_scan. [#265](https://github.com/greenbone/ospd-openvas/pull/265)
 
+[20.08]: https://github.com/greenbone/ospd-openvas/compare/ospd-openvas-20.08...master
```

Comment:
Same for ospd-openvas. It should show a diff of changes included in the release. Therefore I guess it is best to use the diff between the old release branch and the new one.
```suggestion
[20.08]: https://github.com/greenbone/ospd-openvas/compare/ospd-openvas-1.0...ospd-openvas-20.08
```